Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an effect or not.

BGL = 100
sd = 15
n= 36
sbg = 108

hypothesis is 
H0: BGL = 100
Ha: BGL != 100

n is bigger so find z-score

In [6]:
BGL = 100 
sd = 15 
n= 36 
sbg = 108
z = (sbg - BGL)/(15/6) 
z

3.2

if alpha = 0.05 then z = 1.96
if alpha = 0.05 then z = 3.2 > 1.96
Reject NULL Hypothesis
there is 3.2 standard deviations above the BGL mean.